SWIPs/swip-message_to_honey.md

```diff
@@ -48,19 +53,19 @@ This section describes the interaction between the nodes and the oracle in more
   }
 }
 ```
-* After `TTL` expires, nodes will query the price oracle for a new honeyPrices object. 
+* After `TTL` expires, nodes will query the price oracle for a new `honeyPrices` object. 
 * The `TTL` is set as a state-variable in the smart-contract and can be updated.
-* To ensure that it is not possible that a new entry in the `prices` array becomes valid, while not all nodes are updated, the smart-contract or update-policy must ensure that the `validFrom` of new prices must be at least `TTL` seconds in the future.
-* Upon start-up, nodes will look at the contents of their local database and query the price oracle if no `honeyPrices` object is found, or the `TTL` of their cached `honeyPrices` object has expired. 
+* To ensure that it is not possible that a new entry in the `prices` array becomes valid before all nodes are updated, the smart-contract or update-policy must ensure that the `validFrom` of new prices must be at least `TTL` seconds in the future.
```

Comment:
Here suddenly we speak of `validFrom`. Which one are using? Both? Only one?

Reply:
See: [above](https://github.com/Eknir/SWIPs/pull/9/files#r312866558)

Reply:
What do you mean "see above"? Above there is talk of TTL and validUntil, so what is going to be done here with `validFrom`?